.github/workflows/exchain-ut.yml

```diff
@@ -0,0 +1,98 @@
+name: exchain-ut
+
+on:
+  pull_request:
+    branches: [dev]
+
+jobs:
+
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: go mod tidy
+        run:
+          go mod tidy
+      - name: Cache go modules
+        run:
+          make go-mod-cache
+      - name: Build
+        run:
+          make build
+
+  exchain:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: go mod tidy
+        run:
+          go mod tidy
+
+      - name: Run tests with coverage and upload codecov
+        run: |
+          set -ex
+          echo "--> Run tests with coverage:"
+          export VERSION="$(git describe --tags --long | sed 's/v\(.*\)/\1/')"
+          export GO111MODULE=on
+          mkdir -p /tmp/logs /tmp/workspace/profiles
+
+          excluded_packages_expr="(exchain/x/.*/client)|(x/simulation)|(exchain/app/rpc)"
+          included_packages=("./app" "./x/common" "./x/params" "./x/distribution" "./x/staking" "./x/gov" "./x/ammswap" "./x/token" "./x/farm")
```

Comment:
No longer need to run the ut of these two modules: /x/ammswap  and ./x/farm

Reply:
done